common-rendering/src/editorialPalette/border.ts

```diff
@@ -258,6 +258,22 @@ const newsletterSignUpFormDark = (_format: ArticleFormat): string => {
 	return neutral[86];
 };
 
+const tableOfContents = (_format: ArticleFormat): string => {
+	return neutral[86];
+};
+
+const tableOfContentsHover = (_format: ArticleFormat): string => {
+	return neutral[20];
+};
+
+const tableOfContentsDark = (_format: ArticleFormat): string => {
+	return neutral[20];
+};
+
+const tableOfContentsHoverDark = (_format: ArticleFormat): string => {
+	return neutral[86];
+};
+
```

Comment:
given these are duplicates, perhaps we could just have `tableOfContents` and `tableOfContentsDark`?